spec/factories/redirect_messages.rb

```diff
@@ -0,0 +1,29 @@
+require 'govuk_message_queue_consumer/test_helpers/mock_message'
```

Comment:
Could this be a trait on the messages factory?

Reply:
I originally tried that but it got too messy and complicated. Again, I think that redirects are special enough to warrant a distinction. This was also a suggestion from Pablo.